config/notifiers.go

```diff
@@ -306,6 +317,37 @@ func (c *OpsGenieConfig) UnmarshalYAML(unmarshal func(interface{}) error) error
 	return checkOverflow(c.XXX, "opsgenie config")
 }
 
+// VictorOpsConfig configures notifications via VictorOps.
+type VictorOpsConfig struct {
+	NotifierConfig `yaml:",inline"`
+
+	APIKey       Secret `yaml:"api_key"`
+	APIURL       string `yaml:"api_url"`
+	RoutingKey   string `yaml:"routing_key"`
+	MessageType  string `yaml:"message_type"`
+	EntityID     string `yaml:"entity_id"`
```

Comment:
No we don't. In PagerDuty the key is visible to the user as well and we don't care. By default VictorOps creates a random ID for you, so even that it's not that different.

We can think about having something more descriptive in the future but not in the context of this PR. The current group key is also a fairly large digest of things, which would exceed a few thousands characters quite easily.